README.md

```diff
@@ -84,26 +89,26 @@ chart.add_series("series-B", [114, 55, 27, 101, 125, 27, 105])
 chart.render()
 ```
 
-`render` 方法会在本地生成一个 render.html 文件，使用浏览器打开。
+And the generated render.html is rendered as below. Isn't that cool！
 
 <p align="center">
     <img src="https://user-images.githubusercontent.com/19553554/66697904-34a30480-ed0c-11e9-8827-656e2c274ca2.png"  width="85%" />
 </p>
 
-### Notebook 环境
+### Notebook
 
 #### Jupyter Notebook
 
 ![](https://user-images.githubusercontent.com/19553554/66697950-8f3c6080-ed0c-11e9-99db-4337e82bc682.png)
 
 #### JupyterLab
 
-使用 JupyterLab 需要注意
+There are some jupyterlab details what you should pay attention to.
 ```python
-# 1. 在顶部引入声明
+# 1. imoport this on the top.

```

Comment:
~imoport this on the top.~
`import`

Reply:
`top` is matter in this case.

Reply:
it makes no sense for someone who know this language.

Reply:
emmm, you could import the dependency in in function scope.That's not rare in python world.